### PR TITLE
Add tokenlon_trades

### PR DESCRIPTION
**Pull request by 0xkeen**
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

### CODE REVIEW

**Comment by 0xkeen** [comment]
I have read the CLA Document and I hereby sign the CLA

File: models/dex/dex_aggregator_trades.sql [hunk 1]
```diff
@@ -31,6 +31,7 @@ spells with issues, to be excluded in short term:
     ,ref('dodo_aggregator_trades')
     ,ref('zeroex_trades')
     ,ref('kyberswap_aggregator_trades')
+    ,ref('tokenlon_trades')
```
Comment (jeff-dude) [inline models/dex/dex_aggregator_trades.sql:34]: both [this](https://github.com/duneanalytics/spellbook/blob/main/models/tokenlon/ethereum/tokenlon_ethereum_trades.sql) spell and [this](https://github.com/duneanalytics/spellbook/blob/main/models/tokenlon/tokenlon_trades.sql) spell are not pulling `block_month` through, so it's failing when trying to union into the sector spell. can you look to pull that column through and try tests again?

**Review by jeff-dude — CHANGES_REQUESTED** [review]

File: models/dex/dex_aggregator_trades.sql (hunk 1 above)
Comment (0xkeen) [reply models/dex/dex_aggregator_trades.sql:34]: OK, I'll try, thank you

**Comment by jeff-dude** [comment]
please force an update to the failing seed file in the repo so it appears in this PR. we have a bug on accessing seeds in prod right now. any dummy update will do (new line, remove blank line, etc)

**Comment by 0xkeen** [comment]
@jeff-dude pls merge, thanks~

**Review by jeff-dude — APPROVED** [review]
sounds good -- thank you!

**Pull request merged**